fw-child/resources/js/child-functions.js

```diff
@@ -255,6 +255,19 @@ const $ = jQuery;
     }
 
     let pinned_item = null;
+    let isControlbarAbsolute = false; // Track the current position state
+
+    function toggleClassOnControlbarAbsolute() {
+      if (pinned_item && $('#control-bar-tabs').css('position') === 'absolute' && !isControlbarAbsolute) {
+        // The element switched to position 'absolute'
+        $('#control-bar-tabs').addClass('opacity-20');
```

Comment:
Personally, I think `opacity-10` would be more appropriate. If changed, don't forget to also update below.